qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -133,21 +137,23 @@ def __init__(self, parent: 'ZNB', name: str, channel: int,
         else:
             traces = self._parent.ask(f"CONFigure:TRACe:CATalog?")
             if existing_trace_to_bind_to not in traces:
-                raise RuntimeError(f"Trying to bind to {existing_trace_to_bind_to} "
+                raise RuntimeError(f"Trying to bind to"
+                                   f" {existing_trace_to_bind_to} "
                                    f"which is not in {traces}")
             self._tracename = existing_trace_to_bind_to
 
         # map hardware channel to measurement
-        # hardware channels are mapped one to one to qcodes channels
+        # hardware channels are mapped one to one to QCoDeS channels
         # we are not using sub traces within channels.
         if existing_trace_to_bind_to is None:
-            self.write("CALC{}:PAR:SDEF '{}', '{}'".format(self._instrument_channel,
-                                                           self._tracename,
-                                                           self._vna_parameter))
-
-        # source power is dependent on model, but not well documented.
-        # here we assume -60 dBm for ZNB20, the others are set,
-        # due to lack of knowledge, to -80 dBm as of before the edit
+            self.write("CALC{}:PAR:SDEF '{}', '{}'".format(
+                                                    self._instrument_channel,
```

Comment:
hmm, shouldn't the indentation be at the level of `write`? i think it would also make for a better readability to put a newline after `write(` as well.

Reply:
Perhaps its worth the trouble to convert the string formatting to f strings in general?